application/Models/AmazonProduct.cs

```diff
@@ -14,7 +14,10 @@ public class AmazonProduct
 
         //[Required]
         public Int32 Quantity { get; set; }
```

Comment:
it's up to you.I didn't use AmazonProduct class yet so that table doesn't exist in the DB yet.
So far I am just updating the prices in the Product table